core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -158,15 +156,20 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   }
 
   def asProjectOwner(projectName: RawlsBillingProjectName)(op: => Future[PerRequestMessage]): Future[PerRequestMessage] = {
```

Comment:
you probably don't even need this function anymore. It is used to protect add, remove and list members. Sam will do that protection. Just delegate the calls to sam.